project/plugins.sbt

```diff
@@ -10,7 +10,7 @@ addSbtPlugin("org.http4s" % "sbt-http4s-org" % "0.14.7")
 addSbtPlugin("com.github.sbt" % "sbt-native-packager" % "1.9.11")
 addSbtPlugin("io.spray" % "sbt-revolver" % "0.9.1")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.4.3")
-addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.10.1")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.11.0")
```

Comment:
I'm not aware of why we didn't get this update through the Steward pipeline. But this update is required to fix these errors:
```
error] org.scalajs.ir.IRVersionNotSupportedException: Failed to deserialize a file compiled with Scala.js 1.11 (supported up to: 1.8): /home/runner/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-js/scalajs-library_2.13/1.11.0/scalajs-library_2.13-1.11.0.jar:/scala/volatile.sjsir
[error] You may need to upgrade the Scala.js sbt plugin to version 1.11 or later.
```

Reply:
It's because I pinned it in the steward config.
https://github.com/http4s/steward/blob/ea7a4d907d3452e993efa2f377a73c5616c58d14/.github/.scala-steward.conf#L4

I guess it's fine to upgrade. I don't know anymore what to do, and often it's not in our control anyway.

Reply:
Ah, I see. So, then it's up to you to decide about this. I'm not an actual user of JS, honestly.